### ISSUE TITLE
gmime-2.6 out of date? 

### ISSUE THREAD

**Issue by borgauf** [ohwgiles/wemed#8]
gmime-2.6 out of date? My Ubuntu 21.10 only knows about libgmime-3.0-0. No gmime-2.6 available.

```
cmake /home/galaxybeing/opt/wemed
-- Checking for module 'gmime-2.6'
--   No package 'gmime-2.6' found
CMake Error at /usr/share/cmake-3.18/Modules/FindPkgConfig.cmake:545 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindPkgConfig.cmake:733 (_pkg_check_modules_internal)
  CMakeLists.txt:15 (pkg_check_modules)
-- Configuring incomplete, errors occurred!
See also "/home/galaxybeing/opt/wemed/CMakeFiles/CMakeOutput.log".
```

**Comment by ohwgiles** [comment]
Yes, seems that all dependencies have advanced somewhat. I will try to get them all up to date in the coming weeks/months.

**Comment by ohwgiles** [comment]
Solved in 0c2a182a29d9df0100e60ac958274a939af01465
